autograder-core/src/main/java/de/firemage/autograder/core/parallel/AnalysisScheduler.java

```diff
@@ -24,7 +24,7 @@ public AnalysisScheduler(int threads, ClassLoader classLoader) {
         this.waitingAndRunningTaskCount = new AtomicInteger(0);
 
         this.analysisThreads = new ArrayList<>();
-        int actualThreads = threads > 0 ? threads : Math.max(Runtime.getRuntime().availableProcessors() - 2, 1);
+        int actualThreads = threads > 0 ? threads : Math.max(Runtime.getRuntime().availableProcessors() - 2, 1);zz
```

Comment:
`zz`? Looks like a typing error
```suggestion
        int actualThreads = threads > 0 ? threads : Math.max(Runtime.getRuntime().availableProcessors() - 2, 1);
```